### DISCUSSION TITLE
Next JS Development Server unexpectedly shutting down.

### DISCUSSION THREAD

**Discussion by NathLowe** [vercel/next.js#55012] [Help]
### Summary

I was first working on a project with the version 13.4.10 of NextJS then I upgraded to the latest version (at the time **13.4.19**), and since then, every 20 minutes or so the local server I open using "**npm run dev**" just shut down.

Here Is the error I get on the server:
![image](https://github.com/vercel/next.js/assets/89553304/b074d5a9-b400-49d0-8ebd-2de816459fef)

On the browser, I get: **Internal Error 500**

Has anyone encountered this?

**Comment by MarkLyck** [comment]
Reported the same issue here: https://github.com/vercel/next.js/discussions/55033

and this guy reported it 2 weeks ago: https://github.com/vercel/next.js/discussions/54598

Please let me know if you find a solution 🙏

**Comment by NathLowe** [comment]
Yesterday I tried to start my NextJS 13 application on digitalocean but I got the same error.
I built the application then used pm2 to start the application. It worked well at first then suddenly showed theses errors:

![image](https://github.com/vercel/next.js/assets/89553304/7c9dc196-93cb-430e-83d5-66f636b15893)

**Comment by icyJoseph** [comment]
Could it be that your system doesn't allow creation of ports? or rather opening ports in a certain range or at all? https://github.com/vercel/next.js/discussions/51691#discussioncomment-6341657

**Reply by NathLowe** [reply]
I didn't think about it. Thanks, I will check.
